presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -37,6 +37,15 @@ and ``join_type`` is one of
     FULL [ OUTER ] JOIN
     CROSS JOIN
 
+and ``groupby_expr`` is one of
+
+.. code-block:: none
+
+    expression [, ...]
```

Comment:
I think this list is wrong. See the PostgreSQL docs. A non-parenthesised expression should just be `expression` and not have the `...` since this is for a single element, not the full `GROUP BY` list